packages/core/src/Queue.js

```diff
@@ -4,58 +4,60 @@
 
 // Based on https://github.com/petkaantonov/deque
 
-export default function Queue (capPow2) {
-  this._capacity = capPow2 || 32
-  this._length = 0
-  this._head = 0
-}
+export default class Queue {
+  constructor(capPow2) {
```

Comment:
I think it’s cleaner and more visible to the client to have the default value set with the parameter:

```
constructor(capPow2 = 32)
```

Reply:
Agreed.  @davidchase, could you change it in this PR?